bootstrap parser not equivalent to boot parser

The behavior of the (faster) procedural parser is different from the model based parsers at 3 failing tests:

```console
FAILED tests/grammar/parameter_test.py::ParameterTests::test_36_param_combinations - AssertionError: assert 123 == '123'
FAILED tests/grammar/parameter_test.py::ParameterTests::test_numbers_and_unicode - tatsu.exceptions.FailedToken: error: expecting 'None'
FAILED tests/z_bootstrap_test.py::test_08_compile_with_generated - assert '{\n  "direct...": "TATSU"\n}' == '{\n  "rules"...": "TATSU"\n}'
3 failed, 547 passed, 6 skipped, 5 warnings in 26.67s
```

--- a/tatsu/contexts/decorator/rule.py
+++ b/tatsu/contexts/decorator/rule.py
@@ -28,6 +28,6 @@ def rule(*args: Any, **kwargs: Any) -> Callable:
         return __rule_wrapper(func)
 
     def decorator(func: Callable) -> Callable[[Any, Ctx], Any]:
-        return __rule_wrapper(func, *args, *kwargs)
+        return __rule_wrapper(func, *args, **kwargs)
 
     return decorator

--- a/tests/grammar/parameter_test.py
+++ b/tests/grammar/parameter_test.py
@@ -2,10 +2,12 @@
 # SPDX-License-Identifier: BSD-4-Clause
 from __future__ import annotations
 
+import builtins
 import contextlib
 import pathlib
 import sys
 import tempfile
+import types
 import unittest
 
 from tatsu.boot import TatSuParserGenerator
@@ -57,7 +59,7 @@ class ParameterTests(unittest.TestCase):
                 assert_equal('+', p4)
                 return ast
 
-            def rule_keyword(self, ast, k1, k2, k3, k4):
+            def rule_keywords(self, ast, k1, k2, k3, k4):
                 assert_equal('ABC', k1)
                 assert_equal(123, k2)
                 assert_equal('=', k3)
@@ -112,7 +114,13 @@ class ParameterTests(unittest.TestCase):
         semantics = TC36Semantics()
         ast = model.parse('a b c', semantics=semantics)
         self.assertEqual(['a', 'b', 'c'], ast)
-        pythongen(model)
+        code = pythongen(model)
+        module = types.ModuleType('tc36params')
+        module.__file__ = '<tc36params>'
+        exec(builtins.compile(code, module.__file__, 'exec'), module.__dict__)
+        parser = module.RuleArgumentsParser()
+        ast = parser.parse('a b c', semantics=semantics, parseinfo=False)
+        self.assertEqual(['a', 'b', 'c'], ast)
 
     def test_36_unichars(self):
         grammar = """

